.github/workflows/cd.yaml

```
@@ -109,7 +109,7 @@ jobs:
 
       - name: cd/rename-artifacts
         run: |
-          mv dist/*.tar.gz dist/${GITHUB_REPOSITORY#*/}-${GITHUB_REF_NAME}.tar.gz
+          mv dist/*.tar.gz dist/${GITHUB_REF_NAME}.tar.gz
```

```suggestion
          mv dist/*.tar.gz dist/playbooks-${GITHUB_REF_NAME}.tar.gz
```
Shouldn't it be like this to match the old format? Afaiu the `GITHUB_REF_NAME` variable is only the name of the given git tag

---

Aha! Good catch Mario. rebasing and pushing the fix.

---

@mvitale1989  Done with https://github.com/mattermost/mattermost-plugin-playbooks/compare/35594279667c2d936a85114f516b78eca9db7c16..93c90347386be52e60a5ab5996be876ba1bb9a8f